ReactWindows/ReactNative.Shared/UIManager/AccessibleAutomationPeer.cs

```diff
@@ -0,0 +1,104 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using ReactNative.UIManager.Events;
+#if WINDOWS_UWP
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Automation.Peers;
+using Windows.UI.Xaml.Automation.Provider;
+#else
+using System.Windows;
+using System.Windows.Automation.Peers;
+using System.Windows.Automation.Provider;
+#endif
+
+namespace ReactNative.UIManager
```

Comment:
>n [](start = 0, length = 1)

Maybe we should have a comment that we think of `Auto` as `No` for now

Reply:
Actually, Auto != No when determining IsContent and IsControl. I made the fix to those two methods.

---
In reply to: [159568016](https://github.com/reseul/react-native-windows/pull/1#discussion_r159568016) [](ancestors = 159568016)